src/service/fixed-layer.js

```diff
@@ -557,9 +536,18 @@ export class FixedLayer {
 
     // Update `top`. This is necessary to adjust position to the viewer's
     // paddingTop.
-    if (state.top && (state.fixed || (state.sticky && !this.transfer_))) {
```

Comment:
I'm not checking for `state.top`, which can only be truthy if `fixed || sticky`. I can be more explicit if you'd like.